### ISSUE TITLE
Obsolete AUR PKGBUILD

### ISSUE THREAD

**Issue by Firobe** [unikraft/kraftkit#1138]
### Describe the bug

The PKGBUILD for kraftkit's package on AUR [`kraftkit-bin`](https://aur.archlinux.org/packages/kraftkit-bin) expects a `LICENSE.md` file in the release tarball, which is not present since 0.7.1. This causes the install script to fail (see the log). Furthermore, the PKGBUILD does not install `kraftld`.

Hence the PKGBUILD (and probably the bot managing it) should be changed to not try to install `LICENSE.md`, and try to install `kraftld`.

### Steps to reproduce

Attempt installation of `kraftkit-bin` version 0.7.1-1 from the AUR.

### Expected behavior

Successful installation `kraft` and `kraftld`.

### Relevant log output

```shell
==> Making package: kraftkit-bin 0.7.1-1 (mer. 10 janv. 2024 21:14:48)
==> Retrieving sources...
  -> Downloading kraftkit-bin_0.7.1_x86_64.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
100 13.9M  100 13.9M    0     0  14.4M      0 --:--:-- --:--:-- --:--:-- 75.5M
==> WARNING: Skipping verification of source file PGP signatures.
==> Validating source_x86_64 files with sha256sums...
    kraftkit-bin_0.7.1_x86_64.tar.gz ... Passed
:: (1/1) Parsing SRCINFO: kraftkit-bin
==> Making package: kraftkit-bin 0.7.1-1 (mer. 10 janv. 2024 21:14:50)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found kraftkit-bin_0.7.1_x86_64.tar.gz
==> Validating source_x86_64 files with sha256sums...
    kraftkit-bin_0.7.1_x86_64.tar.gz ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Extracting kraftkit-bin_0.7.1_x86_64.tar.gz with bsdtar
==> Sources are ready.
==> Making package: kraftkit-bin 0.7.1-1 (mer. 10 janv. 2024 21:14:52)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Entering fakeroot environment...
==> Starting package()...
install: cannot stat '/home/virgile/.cache/yay/kraftkit-bin/src/LICENSE.md': No such file or directory
==> ERROR: A failure occurred in package().
    Aborting...
```

**Comment by craciunoiuc** [comment]
Hey @Firobe,

Thanks for finding this out, I'll implement your suggestion of replacing the `LICENSE` with `kraftld` and I'll be back with a fix

**Comment by craciunoiuc** [comment]
Did a fix, we'll merge soon, but it'll take effect after a new release (hopefully soon)

**Comment by Firobe** [comment]
Great, will report back when that happens :)

**Comment by Firobe** [comment]
I can confirm the fix works for the latest release (0.7.2-1) on AUR! Thanks again
